lib/statesman/adapters/active_record_queries.rb

```diff
@@ -3,7 +3,7 @@ module Adapters
     module ActiveRecordQueries
       def self.check_missing_methods!(base)
         missing_methods = %i[transition_class initial_state].
-          reject { |_method| base.respond_to?(:method) }
+          reject { |meth| base.respond_to?(meth) }
```

Comment:
```suggestion
          reject { |method| base.respond_to?(method) }
```

Reply:
Yeah, I will do. I avoided `method` because it's an already defined method, I mean it won't have any negative affect but it has as little smell, what do you think?

Reply:
Good point. `m` then? 
just think `meth` could be considered inappropriate

Reply:
Fair enough, will change